catalog/app/containers/Auth/SSOOkta.js

```diff
@@ -49,6 +50,12 @@ export default ({ mutex, next }) => {
     })
     const url = `${oktaDomain}/oauth2/v1/authorize${query}`
     const popup = window.open(url, 'quilt_okta_popup', 'width=300,height=400')
+    const timer = setInterval(() => {
+      if (popup.closed) {
+        clearInterval(timer)
+        handleFailure({ error: 'popup_closed_by_user' })
+      }
+    }, 500)
     popup.focus()
     window.addEventListener('message', ({ origin, data }) => {
```

Comment:
I realized I need to remove the event listener, too. I'll fix it in my PR

Reply:
right